estimagic/decorators.py

```diff
@@ -114,6 +114,19 @@ def wrapper_negative_criterion(*args, **kwargs):
     return wrapper_negative_criterion
 
 
+def negative_gradient(gradient):
```

Comment:
This looks awkward. Why dont you use functools.wraps as an decorator and put the returns below the decorated function. And why dont you return the wrapped function?